sigproc/inc/WireCellSigProc/OmnibusSigProc.h

```diff
@@ -42,7 +42,8 @@ namespace WireCell {
                 const std::string& break_roi_loop1_tag = "break_roi_1st",
                 const std::string& break_roi_loop2_tag = "break_roi_2nd",
                 const std::string& shrink_roi_tag = "shrink_roi", const std::string& extend_roi_tag = "extend_roi",
-                const std::string& mp3_roi_tag = "mp3_roi", const std::string& mp2_roi_tag = "mp2_roi");
+                const std::string& mp3_roi_tag = "mp3_roi", const std::string& mp2_roi_tag = "mp2_roi",
+		std::vector<int> rebase_planes = {0}, int rebase_nbins=200);
```

Comment:
It seems this initialization will rebase plane 0 for all old configurations that do not set any `rebase_planes` option.  Is that desired?  If not, it should be fixed.

Then, just a comment, this constructor has gotten really out of control.  No tests were ever made that make use of all these arguments and if a future test that needs them is made it can (and should) set values via `configure()`.  So,  I suggest not adding more arguments to it and remove these two new ones.  This would also fix the addition of plane=0 (if indeed that is a bug).

You can initialize sane defaults in the declarations.  Eg, 

```c++
   std::vector<int> m_rebase_planes;
   // or if you really meant to add plane 0 then:
   // std::vector<int> m_rebase_planes{0};
   int m_rebase_nbins{200};
```